docs/source/indepth_tutorial/directed-search.ipynb

```diff
@@ -98,9 +98,9 @@
      "name": "stderr",
```

Comment:
<p>Discuss about how and why this dataframe is different from the first one. Just stating that there were only two solutions with <code>max_P</code> below 1 might be enough.</p><p> </p><p>But maybe then also discuss when you would define the constraints in advance in the optimizer, and when you would just run the optimizer without constrains and filter the desired solution afterwards in the results DataFrame. Is there any difference in results from both methods? If so, why and when?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quaquel/EMAworkbench/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>the bigger issue is in my view outside scope. It is a rather complicated topic involving the search space, the structure of the fitness landscape etc. </p>

Reply:
<p>Agreed, just the top one is fine</p>